src/main/java/knu/kproject/controller/ProjectController.java

```diff
@@ -39,15 +39,12 @@ public class ProjectController {
             @ApiResponse(responseCode = "500", description = "서버 오류")
     })
     @PostMapping("")
-    public ResponseEntity<Api_Response<Object>> createProject(@RequestBody PutProjectDto projectDto, @AuthenticationPrincipal Long key) {
+    public ResponseEntity<Api_Response<ProjectDto>> createProject(@RequestBody PutProjectDto projectDto, @AuthenticationPrincipal Long key) {
         UUID projectId = projectService.createProject(projectDto, key);
         Map<String, UUID> projectMap = new HashMap<>();
         projectMap.put("projectId", projectId);
-        return ResponseEntity.ok().body(Api_Response.builder()
-                .code(SuccessCode.INSERT_SUCCESS.getStatus())
-                .message(SuccessCode.INSERT_SUCCESS.getMessage())
-                .result(projectMap)
-                .build());
+        return ApiResponseUtil.createSuccessResponse(
+                SuccessCode.INSERT_SUCCESS.getMessage());
```

Comment:
```suggestion
        return ApiResponseUtil. createResponse(
                SuccessCode.INSERT_SUCCESS.getStatus(),
                SuccessCode.INSERT_SUCCESS.getMessage(),
                projectMap
        );
```
이 경우도 프로젝트를 만들면 만든 프로젝트의 id를 반환하는게 목적이고

Reply:
이것도 돌려두겠습니다